### DISCUSSION TITLE
tabbar

### DISCUSSION THREAD

**Discussion by giscus** [listenzz/RNDemo#8] [Announcements]
# tabbar

React Native 开发指南 | 教程

https://todoit.tech/tabbar.html

**Comment by zyestin** [comment]
https://reactnavigation.org/docs/tab-view/#lazy

大佬，可以加个lazy吗，不然 所有列表一开始就全初始化了

**Reply by listenzz** [reply]
lazy 使用 tab 自身的 index 配合 selectedIndex 自行实现即可
